frontend/src/views/website/runtime/python/operate/index.vue

```diff
@@ -315,7 +183,6 @@ const acceptParams = async (props: OperateRrops) => {
     scripts.value = [];
     if (props.mode === 'create') {
         Object.assign(runtime, initData(props.type));
-        searchApp(null);
         open.value = true;
     } else {
         getRuntime(props.id);
```

Comment:
There doesn't seem to be any significant code differences between the two versions provided. The current version appears to match with the existing functionality, which would include errors and warnings related to missing or invalid props for these fields (i.e., `runtime.name`, `runtime.params` etc.), input validation using `rules`, input field styles such as the el-select dropdown styling in case it's applied outside its default value being set.

For future improvements:

1. Ensure input values validate themselves properly according to their respective validations in `rules`.

2. Make sure that any computed properties (`mode` etc.) are not mutating during runtime state changes if they may introduce bugs or incorrect updates when accessed after an update occurs, like here.

Remember, it's crucial to keep up-to-date documentation and maintainers will appreciate regular feedback on what is working well against breaking changes made without notice due to time constraints. Please ensure everything works as expected across all environments before pushing new functionalities for user adoption.